xla/backends/gpu/codegen/copy.cc

```diff
@@ -44,6 +44,15 @@ limitations under the License.
 namespace xla {
 namespace gpu {
 
+HloInstructionAdaptor SkipOptionalBitcast(HloInstructionAdaptor adaptor) {
```

Comment:
Could this be in an anonymous namespace? Same below?

Reply:
Yes, of course. Thanks!